src/ecrc-frontend/src/components/base/declaration/Declaration.css

```diff
@@ -0,0 +1,33 @@
+.declareTitle {
+  width: 100%;
+  height: 36px;
+  background: inherit;
+  background-color: rgba(56, 89, 138, 1);
+  box-sizing: border-box;
+  border-width: 3px;
+  border-style: solid;
+  -moz-box-shadow: none;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+  font-family: "myriad-pro", "sans serif";
+  font-weight: 400;
+  font-style: normal;
+  color: white;
+  padding-left: 20px;
+  padding-top: 2px;
+  vertical-align: middle;
+}
+
+.declaration-cb {
+  font-size: 14px;
+  color: #333333;
+  text-align: left;
+  line-height: 20px;
+}
+
+.declareSection {
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-top: 20px;
+  padding-bottom: 0px;
```

Comment:
does this need to be provided or is default automatically set to 0px?

Reply:
removed.